lib/nerves_hub/logger.ex

```diff
@@ -94,6 +95,13 @@ defmodule NervesHub.Logger do
     )
   end
 
+  def log_event([:nerves_hub, :devices, :connecting_code_failure], _, metadata, _) do
+    Logger.info("Connecting code failure",
+      event: "nerves_hub.devices.connecting_code_failure",
+      output: metadata[:output]
```

Comment:
```suggestion
      identifier: metadata[:identifier],
      output: metadata[:output]
```
And also include it here 😉